src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,61 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final int DATE_INDEX = 0;
+    private static final int NAME_INDEX = 1;
+    private static final int WORK_HOURS_INDEX = 2;
+    private static final int PAY_PER_HOUR_INDEX = 3;
+    private static final String DATE_FORMAT = "dd.MM.yyyy";
+
+    public static final DateTimeFormatter formatter = DateTimeFormatter.ofPattern(DATE_FORMAT);
```

Comment:
Consider making the formatter a constant field. The naming convention for constants in Java is to use all uppercase with underscores to separate words. So the formatter should be named like FORMATTER.